app/components/post/post.js

```diff
@@ -312,8 +312,8 @@ export default class Post extends PureComponent {
         }
     });
 
-    toggleSelected = (selected) => {
-        if (!getToolTipVisible()) {
+    toggleSelected = (selected, bypass) => {
```

Comment:
I don't understand this function. Why are we checking if the tooltip isn't visible, but we're then bypassing that to mark it selected in some cases with the tooltip? Wouldn't it be much simpler to just only call it when we actually want the selected state to be updated? Maybe that could be included in a comment so we can better understand its purpose when we come back to it later